pkg/crypto/helpers.go

```diff
@@ -121,7 +122,12 @@ func TopicFromCert(cert *x509.Certificate) (string, error) {
 	var oidASN1UserID = asn1.ObjectIdentifier{0, 9, 2342, 19200300, 100, 1, 1}
 	for _, v := range cert.Subject.Names {
 		if v.Type.Equal(oidASN1UserID) {
-			return v.Value.(string), nil
+			var uid = v.Value.(string)
```

Comment:
this will panic if value is not a string. Unlikely but might be worth checking. 

https://golang.org/doc/effective_go.html#interface_conversions
```
str, ok := value.(string)
if ok {
    fmt.Printf("string value is: %q\n", str)
} else {
    fmt.Printf("value is not a string\n")
}
```

Reply:
Wilco